game/lib/game/equipment_and_crafting/equipment.py

```diff
@@ -0,0 +1,40 @@
+from lib.game.items.item import Item
+import pygame
+
+
+class Equipment:
+
+    __item_list = [None, None, None, None, None]
+    __background = pygame.transform.scale(pygame.image.load('config/assets/item_background.png'), (64, 64))
```

Comment:
64 wpisać do zmiennej, coś typu item_resolution. Wtedy jak coś się zmieni to wystarczy jedną zmienną edytować, a nie wszystkie 64 w kodzie.  To samo niżej

Reply:
W sumie widzę, że jest sporo tych 64. Lepiej je dodać do jakiegoś configu i tylko używać w każdej klasie xD